main.sh

```diff
@@ -107,40 +99,40 @@ function xml_dump {
 
 function all_sch { # arg($1:group)
     # is loaded by default so data must be loaded
-    data=($(sed 's/://g' $SCHEDULE))
```

Comment:
eye candy corrections!
`batti | sed 's/ Group [2-7]/\n&/'` might be helpful, can add that in README, 
run batti as the alias itself. for eye candyness.

![screenshot52](https://cloud.githubusercontent.com/assets/1684754/5996072/524ef064-aac9-11e4-8d4a-c20398798826.png)